tests/vmi_configuration_test.go

```diff
@@ -2427,11 +2428,15 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			// ordering:
 			// use a small disk for the other ones
 			containerImage := cd.ContainerDiskFor(cd.ContainerDiskCirros)
-			// virtio - added by NewRandomVMIWithEphemeralDisk
-			vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(containerImage, "echo hi!\n")
-			// sata
-			tests.AddEphemeralDisk(vmi, "disk2", v1.DiskBusSATA, containerImage)
+			// virtio - added by NewCirros
 			// NOTE: we have one disk per bus, so we expect vda, sda
+			vmi = libvmifact.NewCirros(
+				libvmi.WithCloudInitNoCloud(
+					libcloudinit.WithNoCloudUserData("echo hi!\n"),
+				),
```

Comment:
I think this is redundant.

Reply:
I think it checked below.

Reply:
Can you point us to while line you refer to or explain it in more details?
I assumed it is added to expedite the boot on Cirros OS.

Reply:
Didn't understand the original comment properly. fixing.